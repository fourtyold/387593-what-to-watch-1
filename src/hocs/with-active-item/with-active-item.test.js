import React from "react";
import renderer from "react-test-renderer";
import withActiveItem from "./with-active-item.js";

const MockComponent = () => <div />;
const MockComponentWrapped = withActiveItem(MockComponent);

const options = {
  filterGenre: `All genres`,
  setFilter: jest.fn()
};

it(`With-active-genre correctly renders`, () => {
  const tree = renderer.create(<MockComponentWrapped
    filterGenre={options.filterGenre}
    setFilter={options.setFilter}
  />).toJSON();
  expect(tree).toMatchSnapshot();
});
