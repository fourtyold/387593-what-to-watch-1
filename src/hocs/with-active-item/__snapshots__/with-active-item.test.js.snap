// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With-active-genre correctly renders 1`] = `<div />`;
